core/src/main/java/tech/tablesaw/joining/DataFrameJoiner.java

```diff
@@ -59,12 +60,20 @@ public Table inner(Table... tables) {
      */
     public Table inner(boolean allowDuplicateColumnNames, Table... tables) {
         Table joined = table;
-        for (Table table2 : tables) {
-          joined = inner(table2, column.name(), allowDuplicateColumnNames);
+        
+        for (int i = 0; i < tables.length; i++) {
```

Comment:
It's not clear to me why the previous code doesn't work here. can you explain what the issue was?

Reply:
From what I could determine the result wasn't accumulative.  The final value of the "joined" variable returned to the caller was only the join with the last table in the tables list.  There wasn't a junit test for it, and when I added one I saw that it only joined on the last table.

Reply:
Oh, I see why now. It's because `joinInternal` uses `table` as the base table to join off of. Can you add a first parameter `table1` to `joinInternal` and replaces all references to `table` in that function with references to `table1`? Then you can pass `joined` here. And you can keep the counter and probably no longer need the extra logic in `renameColumnsWithDuplicateNames`

Reply:
Knew there was probably a more elegant solution than my brute force one.  Thanks.  I'll work on it.